multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/CheckForServiceBindingOrKeyCreation.java

```diff
@@ -0,0 +1,40 @@
+package org.cloudfoundry.multiapps.controller.process.steps;
+
+import java.time.Duration;
+import java.util.List;
+
+import javax.inject.Named;
+
+import org.cloudfoundry.multiapps.controller.process.Messages;
+import org.cloudfoundry.multiapps.controller.process.variables.Variables;
+
+@Named("checkForServiceBindingOrKeyCreation")
+public class CheckForServiceBindingOrKeyCreation extends TimeoutAsyncFlowableStep {
+
+    @Override
+    protected StepPhase executeAsyncStep(ProcessContext context) throws Exception {
+        boolean isServiceBindingKeyCreationInProgress = context.getVariable(Variables.IS_SERVICE_BINDING_KEY_CREATION_IN_PROGRESS);
```

Comment:
Rename isServiceBindingKeyCreationInProgress and IS_SERVICE_BINDING_KEY_CREATION_IN_PROGRESS to 
isServiceBindingKeyOperationInProgress and IS_SERVICE_BINDING_KEY_OPERATION_IN_PROGRESS as it is not always creation (might be unbding, key deletion, etc)